src/sentry/tasks/autofix.py

```diff
@@ -54,16 +54,26 @@ def generate_issue_summary_only(group_id: int) -> None:
     Generate issue summary WITHOUT triggering automation.
     Used for triage signals flow when event count < 10 or when summary doesn't exist yet.
     """
-    from sentry.seer.autofix.issue_summary import get_issue_summary
+    import sentry_sdk
+
+    from sentry.seer.autofix.issue_summary import _generate_fixability_score, get_issue_summary
 
     group = Group.objects.get(id=group_id)
     logger.info("Task: generate_issue_summary_only, group_id=%s", group_id)
     get_issue_summary(
         group=group, source=SeerAutomationSource.POST_PROCESS, should_run_automation=False
     )
-    # TODO: Generate fixability score here and check for it in run_automation around line 316
-    # That will make sure that even after adding fixability here it's not re-triggered.
-    # Currently fixability will only be generated after 10 events when run_automation is called
+
+    # Generate fixability score
+    with sentry_sdk.start_span(op="ai_summary.generate_fixability_score"):
+        issue_summary = _generate_fixability_score(group)
+
+    if not issue_summary.scores:
+        raise ValueError("Issue summary scores is None or empty.")
+    if issue_summary.scores.fixability_score is None:
+        raise ValueError("Issue summary fixability score is None.")
+
+    group.update(seer_fixability_score=issue_summary.scores.fixability_score)
```

Comment:
seems like this stuff should be in its own `get_fixability_score` function which always checks whether the score already exists before hitting the seer API